src/main/java/com/cloudant/http/interceptors/BasicAuthInterceptor.java

```diff
@@ -41,7 +41,7 @@ public BasicAuthInterceptor(String userinfo, String authHeader) {
     @Override
     public HttpConnectionInterceptorContext interceptRequest(HttpConnectionInterceptorContext
                                                                      context) {
-        context.connection.requestProperties.put("Authorization", String.format("Basic %s",
+        context.connection.requestProperties.put(authHeader, String.format("Basic %s",
```

Comment:
The subclass that does proxy auth needs to make it `Proxy-Authorization` otherwise the code is identical so it seemed to make sense to share it. Maybe there is a more elegant way to change the value, or maybe the constructor that takes the value should be `protected` instead of public?